sample_files/.gitlab-ci.yml

```diff
@@ -0,0 +1,44 @@
+image: vauxoo/odoo-80-image-shippable-auto
+stages:
+  - test
+
+variables:
+    PSQL_VERSION: "9.5"
+    VERSION: "11.0"
+    ODOO_REPO: "odoo/odoo"
+    ODOO_BRANCH: "11.0"
+
+lint:
+  stage: test
+  tags:
+    - lint
+  variables:
+    LINT_CHECK: "1"
+    TESTS: "0"
+  script:
+    - export TRAVIS_BUILD_DIR=$(pwd)
+    - sudo apt-get update && sudo apt-get install -y dos2unix
+    - git clone https://github.com/vauxoo/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
+    - export PATH=${HOME}/maintainer-quality-tools/travis:${HOME}/gitlab_tools:${PATH}
+    - travis_install_nightly
+    - travis_run_tests
+    - travis_after_tests_success || true
+
+test:
+  stage: test
+  tags:
+    - test
+  variables:
+    LINT_CHECK: "0"
+    TESTS: "1"
+  script:
+    - export TRAVIS_BUILD_DIR=$(pwd)
```

Comment:
I think this can get into the main `variables` section with just `TRAVIS_BUILD_DIR: "$CI_PROJECT_DIR"`

Reply:
I got it
Thanks!

Done!

cc @ruiztulio